pzsd_bot/settings.py

```diff
@@ -41,11 +41,18 @@ class _DB(EnvSettings):
     pghost: str = "localhost"
     pgport: int = 5432
     pgdatabase: str = "pzsd"
+    testing: bool = False
+
+    @property
+    def connection_str(self) -> str:
+        if self.testing:
```

Comment:
applications shouldn't be environment aware. you could either pass in the whole connection string or you could rename this to use_sqlite

Reply:
good point i guess. Wouldn't renaming to `use_sqlite` still be considered environment aware though? I think i'll probably just do your first suggestion

Reply:
you could follow django's pattern and use `db_engine` with options `sqlite` and `postgres`

Reply:
So you'd be ok with how I wrote this if I change it to something like this?
```py
@property
def connection_str(self) -> str:
    if self.db_engine == "sqlite":
        ...
    elif self.db_engine == "postgres":
        ...
    else:
        raise SomeError
```

Reply:
yeah that's fine

Reply:
done